extension/redirects.js

```diff
@@ -102,7 +111,7 @@ KNOWN_REDIRECTS.forEach(KNOWN_REDIRECT => {
 
     // We need to generate a few variations on this original pattern
     // 1) support the URL param as the first param
-    newPatterns.push(`${originalPattern}${targetParamKv}`);
+    newPatterns.push(`${originalPattern}*`);
```

Comment:
This is a dirty fix to ensure checkWildcard works. The constructed RegExp didn't work with '?'. Since the parameter is already being checked in another method checking the host here seems to be sufficient.

To reduce redundancy maybe omit newPatterns altogether and push originalPatterns to  `REDIRECT_DATA_BY_TARGET_PARAM[targetParam].patterns` directly?